src/fwupdbase.cpp

```diff
@@ -39,6 +39,12 @@ void FwUpdBase::verify(const fs::path& publicKey, const std::string& hashFunc)
 
 bool FwUpdBase::install(bool reset)
 {
+    if (!files.empty())
+    {
+        // This updater is currently unusable.
```

Comment:
I would say "nothing to install for this updater type"

Reply:
fixed